static/src/javascripts/projects/common/modules/commercial/geo-utils.spec.ts

```diff
@@ -1,26 +1,40 @@
+import type { CountryCode } from '@guardian/libs';
 import {
-	isInUk,
-	isInUsa,
-	isInCanada,
+	_,
+	isInAuOrNz,
 	isInAustralia,
+	isInCanada,
 	isInNewZealand,
-	isInUsOrCa,
-	isInAuOrNz,
 	isInRow,
-	_,
+	isInUk,
+	isInUsa,
+	isInUsOrCa,
 } from 'common/modules/commercial/geo-utils';
 
-let mockCountryCode;
+let mockCountryCode: CountryCode;
 jest.mock('../../../../lib/geolocation', () => ({
 	getCountryCode: jest.fn(() => mockCountryCode),
 }));
 
+type TestCase = {
+	fnName: string;
+	mockCountryCode: CountryCode;
+	expectedUKValue: boolean;
+	expectedUsaValue: boolean;
+	expectedCaValue: boolean;
+	expectedAuValue: boolean;
+	expectedNzValue: boolean;
+	expectedUsOrCaValue: boolean;
+	expectedAuOrNzValue: boolean;
+	expectedRowValue: boolean;
+};
+
 describe('Geolocation Utils', () => {
 	beforeEach(() => {
 		_.resetModule();
 	});
 
-	const testCases = [
+	const testCases: TestCase[] = [
```

Comment:
This is totally optional but it might be possible to avoid defining the `TestCase` type by using an [as const](https://blog.logrocket.com/const-assertions-are-the-killer-new-typescript-feature-b73451f35802/) assertion instead.

Something like

```typescript
const testCases = [
  // ...
] as const;
```

This should tell TypeScript to keep the types as narrow as possible, so `FR` is given the type `FR` and not `string`, so it can be used as a country code in the tests.